app/[...slug]/page.tsx

```diff
@@ -0,0 +1,42 @@
+import { notFound } from "next/navigation"
+import React from "react"
+
+import BasicPageLayout from "@/components/pages/basicPageLayout/BasicPageLayout"
+import { fetcher } from "@/lib/graphql/fetchers/dpl-cms.fetcher"
+import {
+  GetPageByPathDocument,
+  GetPageByPathQuery,
+  NodeGoPage,
+} from "@/lib/graphql/generated/dpl-cms/graphql"
+
+async function page(props: { params: Promise<{ slug: string[] }> }) {
+  const params = await props.params
+
+  const { slug } = params
+
+  const data = await fetcher<GetPageByPathQuery, { path: string }>(GetPageByPathDocument, {
+    path: slug.join("/"),
+  })()
+
+  const routeType = data.route?.__typename
+
+  if (routeType === "RouteRedirect") {
+    // TODO: implement redirect
+    return
+  }
+
+  if (routeType === "RouteExternal") {
+    // TODO: implement external route redirect
+    return
+  }
```

Comment:
In general I am not fan of code stubs if they are not needed. Could we maybe just write one todo where we write that we need to take care of these two cases?
Consider creating a Jira task for it too.